scripts/kotlin-lint-check.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+echo "********************************"
+echo "Checking code formatting"
+echo "********************************"
+
+ktlint --android app/src/**/*.kt data/src/**/*.kt domain/src/**/*.kt testing/src/**/*.kt utility/src/**/*.kt
```

Comment:
I realized this quite late, but this requires ktlint to be in the user's path (which may not always be the case for everyone).

I think we should actually set up ktlint for users as part of the setup script. It seems like we can make it work for both Unix & Windows by executing ktlint as a Jar file directly (e.g. 'java -jar') per its documentation. We just need to download it to a utility outside of the oppia-android repository.

On web, we suggest people put the oppia repo in a folder called 'opensource'. I think we should continue that pattern here, that way we can introduce a new directory structure that simplifies this:
- opensource/oppia-android
- opensource/oppia-android-tools (this isn't a repo and is where things like ktlint will go)

What do you think?

Reply:
Sounds good, I am also using the same pattern 
<img width="203" alt="Screenshot 2020-12-10 at 15 42 11" src="https://user-images.githubusercontent.com/25057618/101758442-59f0f580-3afe-11eb-95f9-00ef522b262f.png">

Reply:
I am thinking to copy start.py from Oppia so that I will get a conditional statement like `common.is_linux_os()` `elif common.is_mac_os()` and else `windows` 
So, the flow will be -> download release file, and run the commands with jar fil. 
Windows -> `java -jar ktlint`
Linux -> `java -jar ktlint`
and for mac, I am thinking to do `brew install ktlint` or should we go with the jar file option only and run commands like `java -jar ktlint`?

https://github.com/oppia/oppia/blob/0068ec7614c31a9389739379943ccdfba0f7f6e0/scripts/start.py#L174

Reply:
Done, using ktlint executable file.

Reply:
![Screenshot 2020-12-18 at 16 08 21](https://user-images.githubusercontent.com/25057618/102605431-458aa980-414b-11eb-91c8-3e0f29333b70.png)